src/Pos/WorkMode.hs

```diff
@@ -149,8 +108,8 @@ type RawRealMode ssc =
     SscHolder ssc (
     NtpSlotting (
     SlottingHolder (
-    ContextHolder ssc (
     DBHolder (
+    ContextHolder ssc (
```

Comment:
The difference is that now `MonadDB = MonadReader NodeDBs`.

Both `ContextHolder` and `DBHolder` implement `MonadReader`, so the outermost layer wins.

If `ContextHolder` wins, the `DBHolder` beneath it is useless.